### PR TITLE
Content Moderator- Set language as a non-required field for Screen Text API 

### PR DESCRIPTION
**Pull request by v-sodsou**
## Description

https://github.com/Azure/azure-rest-api-specs/pull/2884
---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [x] Please add REST spec PR link to the SDK PR
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/SDKs/CognitiveServices/dataPlane/Vision/ContentModerator/ContentModerator.Tests/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -32,5 +32,5 @@
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("0.11.0.0")]
-[assembly: AssemblyFileVersion("0.11.0.0")]
+[assembly: AssemblyVersion("0.12.0.0")]
+[assembly: AssemblyFileVersion("0.12.0.0")]
```
Comment (dsgouda) [inline src/SDKs/CognitiveServices/dataPlane/Vision/ContentModerator/ContentModerator.Tests/Properties/AssemblyInfo.cs]: Please do not update the AssemblyVersion here

**Review by dsgouda — CHANGES_REQUESTED** [review]
Please create a generate.ps1 file similar to [this](https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/CognitiveServices/dataPlane/Language/Language/Language/generate.ps1) and regenerate the code.

**Comment by v-sodsou** [comment]
Regarding "Please create a generate.ps1 file similar to this and regenerate the code." I have already regenerated the code and committed the files changed along with metadata file(.txt file) while got generated when I ran generate.cmd command.

**Comment by dsgouda** [comment]
I understand, we are slowly migrating towards using a generate.ps1 script instead of a cmd. You can do it later if you wish but we are eventually going to get there

**Comment by v-sodsou** [comment]
@dsgouda 
 When I ran generateTool.ps1 , I see log file generated in metadata folder but not ps1 file in my sdk folder.
Could you provide the instructions on powershell script to be run.

**Comment by dsgouda** [comment]
You need to create a `generate.ps1` file similar to the one linked and put the proper relative paths and other args as in the example

**Comment by v-sodsou** [comment]
@dsgouda 

Hi , I have added the generate.ps1 file. could you please review it.

Thanks
Sonali Dsouza

**Comment by dsgouda** [comment]
@v-sodsou looks like the changes to package versions are no longer in the PR, please take a look

**Comment by v-sodsou** [comment]
@dsgouda 
I have added the version changes.Could you please review it.
Thanks
Sonali Dsouza

**Review by dsgouda — APPROVED** [review]
LGTM subject to builds passing

**Pull request merged**